modules/oe_list_pages_address/src/Plugin/facets/processor/FormatCountryCodeProcessor.php

```diff
@@ -64,9 +64,15 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function build(FacetInterface $facet, array $results) {
-    // Loop over all results.
+    // Loop over all results and try to determine the country.
     foreach ($results as $i => $result) {
-      $country = $this->countryRepository->get($result->getRawValue());
+      try {
+        $country = $this->countryRepository->get($result->getRawValue());
```

Comment:
Here we needed to make sure the site didn't crash if the country was not found based on a wrong country code.